python/graphscope/__init__.py

```diff
@@ -16,6 +16,13 @@
 # limitations under the License.
 #
 
+import sys
+
+if sys.version_info[:2] < (3, 7):
+    m = "Python 3.7 or later is required for GraphScope (%d.%d detected)."
+    raise ImportError(m % sys.version_info[:2])
+del sys
```

Comment:
No need for that. We won't distribute wheels that built with python 3.6 anymore so that users have no change to installed the wheels under python 3.6.